### PR TITLE
Add list of Z-Wave Command Classes supported in Home Assistant

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced a comprehensive table in the documentation detailing Z-Wave Command Classes, including their required versions and security measures.

- **Documentation**
	- Enhanced clarity and usability of the Z-Wave integration documentation for Home Assistant users.

### CODE REVIEW

File: source/_integrations/zwave_js.markdown [hunk 1]
```diff
@@ -966,3 +966,28 @@ In Home Assistant, it is currently not possible to set the Z-Wave controller int
 ### Including / excluding a controller in an existing network using [classic inclusion](#classic-inclusion-versus-smartstart)
 
 A Z-Wave controller that manages an empty network can also join a different network and act as a secondary controller there. However, with Home Assistant, this is not possible. Home Assistant does not allow the Z-Wave controller to join another network, because Home Assistant acts as the central hub.
+
+## Z-Wave Command Classes supported in Home Assistant
+
+The following table lists Command Classes that are supported in Home Assistant, together with their version and their required
+Security Class.
```
Comment (AlCalzone) [inline source/_integrations/zwave_js.markdown]: This is tricky...

In Z-Wave terms, a **supported** command class (CC) means that another Z-Wave device can send HA controlling commands of the same CC, and will get a correct response. Those have to be documented.

However users likely understand something else under **supported**, namely which CCs they can interact with in their devices. That list is much, much longer, does not have to be documented (and IMO should not).

I think we need to distinguish between those meanings here. I'm not 100% sure how though. Maybe
 ```suggestion
The following table lists Command Classes that Home Assistant will respond to when queried by other devices, together with the implemented version and required security class:
```

File: source/_integrations/zwave_js.markdown [hunk 2]
```diff
@@ -966,3 +966,28 @@ In Home Assistant, it is currently not possible to set the Z-Wave controller int
...
+
+| Command Class                 | Version Required | Security Class  |
```
Comment (AlCalzone) [inline source/_integrations/zwave_js.markdown]:
```suggestion
| Command Class                 | Version | Security Class  |
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: source/_integrations/zwave_js.markdown [hunk 3]
```diff
@@ -966,3 +966,27 @@ In Home Assistant, it is currently not possible to set the Z-Wave controller int
 ### Including / excluding a controller in an existing network using [classic inclusion](#classic-inclusion-versus-smartstart)
 
 A Z-Wave controller that manages an empty network can also join a different network and act as a secondary controller there. However, with Home Assistant, this is not possible. Home Assistant does not allow the Z-Wave controller to join another network, because Home Assistant acts as the central hub.
+
+## Z-Wave Command Classes supported in Home Assistant
+
+The following table lists Command Classes that Home Assistant will respond to when queried by other devices, together with the implemented version and required security class:
```
Comment (c0ffeeca7) [inline source/_integrations/zwave_js.markdown]:
```suggestion
The following table lists the Command Classes together with the implemented version and required security class. These are the Command Classes that Home Assistant will respond to when queried by other devices. 
```

File: source/_integrations/zwave_js.markdown [hunk 4]
```diff
@@ -966,3 +966,27 @@ In Home Assistant, it is currently not possible to set the Z-Wave controller int
 ### Including / excluding a controller in an existing network using [classic inclusion](#classic-inclusion-versus-smartstart)
 
 A Z-Wave controller that manages an empty network can also join a different network and act as a secondary controller there. However, with Home Assistant, this is not possible. Home Assistant does not allow the Z-Wave controller to join another network, because Home Assistant acts as the central hub.
+
+## Z-Wave Command Classes supported in Home Assistant
+
+The following table lists the Command Classes together with the implemented version and required security class. These are the Command Classes that Home Assistant will respond to when queried by other devices. 
+
+| Command Class                 | Version | Security Class  |
```
Comment (c0ffeeca7) [inline source/_integrations/zwave_js.markdown]:
```suggestion
| Command Class                 | Version        | Security Class  |
```

File: source/_integrations/zwave_js.markdown [hunk 5]
```diff
@@ -966,3 +966,27 @@ In Home Assistant, it is currently not possible to set the Z-Wave controller int
 ### Including / excluding a controller in an existing network using [classic inclusion](#classic-inclusion-versus-smartstart)
 
 A Z-Wave controller that manages an empty network can also join a different network and act as a secondary controller there. However, with Home Assistant, this is not possible. Home Assistant does not allow the Z-Wave controller to join another network, because Home Assistant acts as the central hub.
+
+## Z-Wave Command Classes supported in Home Assistant
+
+The following table lists the Command Classes together with the implemented version and required security class. These are the Command Classes that Home Assistant will respond to when queried by other devices. 
+
+| Command Class                 | Version        | Security Class  |
```
Comment (c0ffeeca7) [inline source/_integrations/zwave_js.markdown]:
```suggestion
| Command Class                 | Version         | Security Class  |
```

File: source/_integrations/zwave_js.markdown [hunk 6]
```diff
@@ -966,3 +966,27 @@ In Home Assistant, it is currently not possible to set the Z-Wave controller int
 ### Including / excluding a controller in an existing network using [classic inclusion](#classic-inclusion-versus-smartstart)
 
 A Z-Wave controller that manages an empty network can also join a different network and act as a secondary controller there. However, with Home Assistant, this is not possible. Home Assistant does not allow the Z-Wave controller to join another network, because Home Assistant acts as the central hub.
+
+## Z-Wave Command Classes supported in Home Assistant
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_js.markdown]:
```suggestion
## Z-Wave Command Classes supported in Home Assistant when queried
```

File: source/_integrations/zwave_js.markdown [hunk 7]
```diff
@@ -966,3 +966,27 @@ In Home Assistant, it is currently not possible to set the Z-Wave controller int
...
+
+The following table lists the Command Classes together with the implemented version and required security class. These are the Command Classes that Home Assistant will respond to when queried by other devices. 
+
+| Command Class                 | Version         | Security Class  |
```
Comment (c0ffeeca7) [inline source/_integrations/zwave_js.markdown]:
```suggestion
| Command Class                 | Version          | Security Class  |
```

File: source/_integrations/zwave_js.markdown [hunk 8]
```diff
@@ -966,3 +966,27 @@ In Home Assistant, it is currently not possible to set the Z-Wave controller int
 ### Including / excluding a controller in an existing network using [classic inclusion](#classic-inclusion-versus-smartstart)
 
 A Z-Wave controller that manages an empty network can also join a different network and act as a secondary controller there. However, with Home Assistant, this is not possible. Home Assistant does not allow the Z-Wave controller to join another network, because Home Assistant acts as the central hub.
+
+## Z-Wave Command Classes supported in Home Assistant when queried
```
Comment (c0ffeeca7) [inline source/_integrations/zwave_js.markdown]:
```suggestion
## Z-Wave Command Classes Home Assistant responds to when queried
```
or maybe like this?

Comment (MartinHjelmare) [reply source/_integrations/zwave_js.markdown]: Yeah, that's good.

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**